synapse/handlers/e2e_room_keys.py

```diff
@@ -351,6 +351,7 @@ def get_version_info(self, user_id, version=None):
                     raise
 
             res["count"] = yield self.store.count_e2e_room_keys(user_id, res["version"])
+            res["etag"] = str(res["etag"])
```

Comment:
I checked briefly on Friday and this matches the other places we call `get_e2e_room_keys_version_info`.